VictorovaHomework1.cpp

```diff
@@ -0,0 +1,56 @@
+#include <cmath>
+#include <iostream>
+
+int main() {
+    constexpr auto coef1{ 4 };
+    constexpr auto coef2{ 2 };
+    auto a{ 0 };
+    auto b{ 0 };
+    auto c{ 0 };
+    auto D{ 0.0 };
+    auto x1{ 0.0 };
+    auto x2{ 0.0 };
+
+    std::cout << "Please enter coefficient a in range [2...365] -->" << std::endl;
+    std::cin >> a;
+    if (a >= 2 && a <= 365) {
+        std::cout << "You entered a correct number" << std::endl;
+    }
+    else {
+        std::cout << "You entered incorrect number, try to enter the number again"
+            << std::endl;
```

Comment:
т.е. мы ввели некоректное значение аргумента и пошли дальше вводить другой аргумент?
это некорекнтно, вы должны если пользователь ввел некоректное значение, как вариант прекратить выполнение вашей програмы, т.к. пользователь ввел некоректные данные.
```suggestion
      else {
        std::cout << "You entered incorrect number, try to enter the number again"
            << std::endl;
         return -1;
      }
```